collections/Don/MS_Don_e_250.xml

```diff
@@ -71,10 +76,441 @@
+                              <height>110</height>
+                              <width>70</width>
+                           </dimensions> </layout>
+                        </layoutDesc>
+                     </objectDesc>
+                     <handDesc hands="2">
+                        <handNote script="cursiva">Cursive (bastarda). Two main hands, fols. 1r-32v and (a later hand) 32v-34v.</handNote>
+                     </handDesc>
+                     <decoDesc>
+                        <decoNote type="flourInit">Penwork frogspawn decoration on two initials (fols 1r and 16r).</decoNote>
+                        <decoNote type="colInit">(fols 1r–32v) One- to four-line initials in red marking text segments. (fols 32v–33r) Simple one- to two-line initials. </decoNote>
+                        <decoNote type="rubrication">(fols 1r–32v) Red headings. The names of the Evangelists in red, Jesus’ name always underlined in red in the gospel harmony. Majuscules at the beginning of phrases stroked in red. (fols 32v–33r) A red heading, one heading and one passage underlined in red. </decoNote>
+                     </decoDesc>
+                  </physDesc>
+                  <history>
+                     <origin>
+                        <origDate calendar="Gregorian" notAfter="1500" notBefore="1440">15th century, middle or second half</origDate>
+                        <origPlace>
+                           <country key="place_7000084">German</country>, <region>west</region>
+                        </origPlace>
+                     </origin>
+                  </history>
+               </msPart>
+               <msPart xml:id="MS_Don_e_250-partB">
+                  <msIdentifier>
+                     <altIdentifier type="partial">
+                        <idno type="part">MS. Don. e. 250 - part B (fols. 35-119)</idno>
+                     </altIdentifier>
+                  </msIdentifier>
+                  <msContents>
+                     <textLang mainLang="la">Latin</textLang>
+                     <msItem n="7" xml:id="MS_Don_e_250-partBi-item7">
+                        <locus>(fols 35r–36v)</locus>
+                        <rubric> Modus Meditandi ihesus Memoriale passionis dominice</rubric>
+                        <incipit>Sedule mediteris eius passionem et infigar cordi tuo sicut ipse iugiter licet et fert infixa wlnerum</incipit>
+                        <explicit>graciarum plenitudinem / et a cunctis  malis imminentibus securam protectionem</explicit>
+                     </msItem>
+                     <msItem n="8" xml:id="MS_Don_e_250-partB-item8">
+                        <locus>(fols 36v–38r)</locus><!-- recte 39r? -->
+                        <author key="person_4949605"><supplied>LUDOLPH OF SAXONY</supplied></author>
+                        <title key="work_3016"><supplied>Vita Jesu Christi</supplied></title>
```

Comment:
Are you wanting to experiment with using `<supplied>` with `<author>` and `<title>`, or is this left over from the original description?

Reply:
convention in the original description - common convention in descriptions from the German speaking world and also in descriptions of Greek mss. in my experience. It has the virtue of making explicit whether an author/title is explicitly named in the rubric.